mysql/lib/dialect/mysql2_driver_dialect.ts

```diff
@@ -52,4 +52,6 @@ export class MySQL2DriverDialect implements DriverDialect {
   getAwsPoolClient(props: PoolOptions): AwsPoolClient {
     return new AwsMysqlPoolClient(props);
   }
+
+  setupInitialProperties(props: Map<string, any>) {}
```

Comment:
Should we notify user that this feature isn't supported by mysql driver if the user provides tcpKeepAlive configuration?

Reply:
I've added a note in the UsingTheNodejsWrapper.md file, do you think that's sufficient?